app/src/apdu_handler.c

```diff
@@ -52,7 +52,7 @@ __Z_INLINE void handleGetAddress(volatile uint32_t *flags, volatile uint32_t *tx
 
     if (requireConfirmation) {
         view_review_init(addr_getItem, addr_getNumItems, app_reply_address);
-        view_review_show();
+        view_review_show(0x03);
```

Comment:
what is the meaning of 0x03?
Can we add some DEFINES in zxlib so the meaning is more clear?